### ISSUE TITLE
feature(core.completion): add support for blink.cmp

### ISSUE THREAD

**Issue by Frestein** [nvim-neorg/neorg#1603]
### Issues

- [X] I have checked [existing issues](https://github.com/vhyrro/neorg/issues?q=is%3Aissue) and there are no existing ones with the same request.

### Feature description

I’m using [blink.cmp](https://github.com/Saghen/blink.cmp) for completion, but I noticed it’s not supported. Could you please add support for blink.cmp?

### Help

No

**Comment by max397574** [comment]
I'd recommend to use neorg-interim-ls anyways
that just creates and lsp server for the completions which is completion engine independent

https://github.com/benlubas/neorg-interim-ls

**Comment by Frestein** [comment]
> I'd recommend to use neorg-interim-ls anyways that just creates and lsp server for the completions which is completion engine independent
> 
> https://github.com/benlubas/neorg-interim-ls

Looks interesting! Thanks!

**Comment by benlubas** [comment]
Yeah there's little to no reason we ever need to support a specific completion engine now that interim-ls exists.

PRs would be accepted though!

**Comment by paul-ohl** [comment]
I think it would make sense to either update the "modules/completion" documentation or add the support, because blink.cmp is now in [kickstart.nvim](https://github.com/nvim-lua/kickstart.nvim/blob/3338d3920620861f8313a2745fd5d2be39f39534/init.lua#L781) so I think quite a few people are going to run into this issue.
I'm glad to edit the Wiki if the first option is approved.
